app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java

```diff
@@ -213,6 +213,43 @@ public Course getCourse(int id) {
         return course;
     }
 
+    // Get courses by list of ids
+    public List<Course> getCourses(String[] ids) {
```

Comment:
why would we want to get courses by id's? Its not like we have a list of id's readily available, or do we?